apis/go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/stretchr/testify v1.10.0
 	k8s.io/api v0.31.3
 	k8s.io/apimachinery v0.31.3
-	sigs.k8s.io/controller-runtime v0.19.2
+	sigs.k8s.io/controller-runtime v0.19.3
```

Comment:
This is bumped by dependabot already

Reply:
Thanks for the head up . I'll rebase once that PR merges.